src/main/java/ProjectOcean/Controller/CourseListIconController.java

```diff
@@ -0,0 +1,54 @@
+package ProjectOcean.Controller;
+
+import ProjectOcean.Model.CoursePlanningSystem;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Text;
+
+import java.io.IOException;
+import java.util.UUID;
+
+public class CourseListIconController extends VBox {
+
+    @FXML
+    private Text courseCodeText;
+
+    @FXML
+    private Text courseNameText;
```

Comment:
Change to make @FXML annotation to be on the same row as the variable declaration
Example: 

@FXML private Text courseCodeText;